local-setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+echo "Creating a test user in localstack"
+awslocal --endpoint-url=http://localstack:4566 iam create-user --user-name test
+
+echo "Creating test credentials in localstack and saving them to .env.local file"
+awslocal --endpoint-url=http://localstack:4566 iam create-access-key --user-name test --output json | jq -r '.AccessKey | "AWS_ACCESS_KEY_ID=\(.AccessKeyId)\nAWS_SECRET_ACCESS_KEY=\(.SecretAccessKey)"' >> .env.local
+
+echo "Creating KMS key in localstack"
+python setup.py create-key-and-csr 'CN=John Smith,O=C2PA Python Demo'
```

Comment:
We could fish `'CN=John Smith,O=C2PA Python Demo'` from a setup env file var value as a next step

Reply:
If we do, it should be as a Name and an Organization field and we can construct the right string with CN= & O=

Reply:
We can follow up with that.